src/apps/components/juttle-editor.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import AceEditor from 'juttle-react-ace';
```

Comment:
I'm kind of questioning what we really get for using `react-ace`. Seems like it kind of just obscures a lot the ability to use the full range of customization available on ace. I'm kind of leaning towards just using `brace` instead.